job-dsls/jobs/osbs/operators_osbs_seed_cloud_jobs.groovy

```diff
@@ -156,11 +156,7 @@ prodComponent.each { Component ->
                           
                           operator_branch = getOperatorBranch(PROD_COMPONENT)
                           
-                          // Checking out from git the Operator repo
-                          sh 'git config --global user.email $GIT_EMAIL'
-                          sh 'git config --global user.name $GIT_AUTHOR'
-                          githubscm.getRepositoryScm(operator_repo_name, GITHUB_ORG_UNIT, operator_branch)
-                          
+                          checkout(githubscm.resolveRepository(operator_repo_name, GITHUB_ORG_UNIT, operator_branch, false))
```

Comment:
Just as reminder, `GITHUB_ORG_UNIT` must be the repository owner e.g., `kiegroup`.